utilities/blob_db/blob_db_test.cc

```diff
@@ -664,6 +664,7 @@ TEST_F(BlobDBTest, GCOldestSimpleBlobFileWhenOutOfSpace) {
   Options options;
   options.env = mock_env_.get();
   BlobDBOptions bdb_options;
+  bdb_options.is_fifo = true;
```

Comment:
This test, BlobDBTest.GCOldestSimpleBlobFileWhenOutOfSpace, already fails with the new code I added. And I think that is right ... the test shouldn't have passed in the first place as the size is going beyond the configured  `blob_dir_size` size.